files/image_config/rsyslog/rsyslog.d/00-sonic.conf

```diff
@@ -1,15 +1,12 @@
 ## Quagga rules
 
-if $programname == ["bgp#quagga",
-                    "bgp#watchquagga",
-                    "bgp#zebra"]
-    then {
-    /var/log/quagga/zebra.log
+if re_match($programname, "bgp[0-9]*#(frr|zebra)") then {
+    /var/log/frr/zebra.log
```

Comment:
Do we plan to give each BGP implementation a separate folder, or should we put it all to /var/log/bgpd/*?

Reply:
The plan was to log it to a single file .. but will have the bgp<n> namespace identifier to identify from which namespace/docker frr instance it is from.
eg:
INFO **bgp0**#bgpd[67]: %ADJCHANGE: neighbor 10.1.0.2(str-n3164-acs-1) in vrf default Up
INFO **bgp1**#bgpd[67]: %ADJCHANGE: neighbor 10.1.0.6(str-n3164-acs-1) in vrf default Up

For the teamd too we have a single file ( /var/log/teamd.log), where we differentiate based on the namespace/docker instance teamd<n>